api/v1beta2/flowcollector_types.go

```diff
@@ -883,15 +817,118 @@ type SASLConfig struct {
+
+	//+kubebuilder:validation:Minimum=1025
+	//+kubebuilder:validation:Maximum=65535
+	//+kubebuilder:default:=2055
+	// Port of the flow collector (host port).
+	// By convention, some values are forbidden. It must be greater than 1024 and different from
+	// 4500, 4789 and 6081.
+	Port int32 `json:"port,omitempty"`
+
+	//+kubebuilder:validation:Minimum=1
+	//+kubebuilder:validation:Maximum=65535
+	//+kubebuilder:default:=8080
+	// `healthPort` is a collector HTTP port in the Pod that exposes the health check API
+	HealthPort int32 `json:"healthPort,omitempty"`
+
+	//+kubebuilder:validation:Minimum=0
+	//+kubebuilder:validation:Maximum=65535
+	//+optional
+	// `profilePort` allows setting up a Go pprof profiler listening to this port
+	ProfilePort int32 `json:"profilePort,omitempty"`
+
+	//+kubebuilder:default:=true
+	// `enableKubeProbes` is a flag to enable or disable Kubernetes liveness and readiness probes
+	EnableKubeProbes *bool `json:"enableKubeProbes,omitempty"`
+
+	//+kubebuilder:default:=true
+	// `dropUnusedFields` allows, when set to `true`, to drop fields that are known to be unused by OVS, to save storage space.
+	DropUnusedFields *bool `json:"dropUnusedFields,omitempty"`
+
+	//+kubebuilder:default:="30s"
+	// +optional
+	// `conversationHeartbeatInterval` is the time to wait between "tick" events of a conversation
+	ConversationHeartbeatInterval *metav1.Duration `json:"conversationHeartbeatInterval,omitempty"`
+
+	//+kubebuilder:default:="10s"
+	// +optional
+	// `conversationEndTimeout` is the time to wait after a network flow is received, to consider the conversation ended.
+	// This delay is ignored when a FIN packet is collected for TCP flows (see `conversationTerminatingTimeout` instead).
+	ConversationEndTimeout *metav1.Duration `json:"conversationEndTimeout,omitempty"`
+
+	//+kubebuilder:default:="5s"
+	// +optional
+	// `conversationTerminatingTimeout` is the time to wait from detected FIN flag to end a conversation. Only relevant for TCP flows.
+	ConversationTerminatingTimeout *metav1.Duration `json:"conversationTerminatingTimeout,omitempty"`
+
+	//+kubebuilder:default="1s"
+	// `minBackoff` is the initial backoff time for client connection between retries.
+	MinBackoff *metav1.Duration `json:"minBackoff,omitempty"` // Warning: keep as pointer, else default is ignored
+
+	//+kubebuilder:default="5s"
+	// `maxBackoff` is the maximum backoff time for client connection between retries.
+	MaxBackoff *metav1.Duration `json:"maxBackoff,omitempty"` // Warning: keep as pointer, else default is ignored
+
+	//+kubebuilder:validation:Minimum=0
+	//+kubebuilder:default:=2
+	// `maxRetries` is the maximum number of retries for client connections.
+	MaxRetries *int32 `json:"maxRetries,omitempty"`
+
+	//+kubebuilder:default:={"app":"netobserv-flowcollector"}
+	// +optional
+	// `staticLabels` is a map of common labels to set on each flow.
+	StaticLabels map[string]string `json:"staticLabels"`
```

Comment:
I'm fine with your suggestion to move flp-related loki config under processor;
For those 4 fields I guess they should be prefixed with "Loki" like you did for the others

Reply:
Renamed while rebasing https://github.com/netobserv/network-observability-operator/compare/3ae36b13d67536bd3a8ca834c04813f9c72aad31..93e67291a0ab7be328dbe506037dd39c8e5b852a